source/2020-01-10-the-ember-times-issue-130.md

```diff
@@ -187,6 +199,9 @@ building modern Ember apps. In this regard we highly recommend the following lea
 - [Rewriting Apps in Ember Octane](https://crunchingnumbers.live/2019/12/23/rewriting-apps-in-ember-octane/) by [Isaac Lee (@ijlee2)](https://github.com/ijlee2)
 - [The Ember Octane vs. Classic Ember Cheat Sheet](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/)
 
+
+> Octane provides a very gentle on-ramp and lets you learn and grow into the framework as the needs arise in your app ... I’m excited to see all of theses pieces coming together and finally ready to teach the version of ember we had envisioned. Check out the new tutorial to see the difference for yourself! ~ _Godfrey Chan [(@chancancode)](https://github.com/chancancode)_
```

Comment:
What do you think about using the full quote here? Since we might not have time to re-review the PR with all folks who gave a quote, I think it'd be great to use the full ones where possible (except for the intro, which should be short and sweet of course!)

```suggestion
> Octane provides a very gentle on-ramp and lets you learn and grow into the framework as the needs arise in your app. You start with HTML templates, then extract pieces into template-only components for clarity, add arguments for reusability, add a js class to add behavior,  add a service for shared functionalities, install an addon for more cool stuff, etc. I’m excited to see all of theses pieces coming together and finally ready to teach the version of ember we had envisioned. Check out the new tutorial to see the difference for yourself! ~ _Godfrey Chan [(@chancancode)](https://github.com/chancancode)_
```